Formula/libmarpa.rb

```diff
@@ -0,0 +1,42 @@
+class Libmarpa < Formula
+  desc "Marpa parse engine C library -- STABLE"
+  homepage "https://jeffreykegler.github.io/Marpa-web-site/libmarpa.html"
+  url "https://github.com/jeffreykegler/libmarpa/archive/refs/tags/v8.6.2.tar.gz", using: :nounzip
```

Comment:
Why do we need to use `:nounzip`?

Reply:
To reproduce the build directory structure recommended by the library's HOWTO document